### PR TITLE
feat: Support Axe-core context param

### PR DESCRIPTION
**Pull request by J-Kallunki**
Add Axe-core context to reactAxe-parameters.

Closes issue:

#41 

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for security

### CODE REVIEW

**Comment by J-Kallunki** [comment]
@WilcoFiers So to get `npm run test` pass needs to `cd example` && `npm install styled-components`. Do you know anything about that?

**Comment by J-Kallunki** [comment]
So I tested it to be working by changing [`example/src/example.js:21`](https://github.com/dequelabs/react-axe/blob/develop/example/src/example.js#L21) to
`axe(React, ReactDOM, 1000, axeConf, {include: [['header']]});`

Might need some help to get cypress test created for the `context`-parameter.

File: index.js [hunk 1]
```diff
@@ -237,10 +241,11 @@ function addComponent(component) {
   }
 }
 
-var reactAxe = function reactAxe(_React, _ReactDOM, _timeout, conf) {
+var reactAxe = function reactAxe(_React, _ReactDOM, _timeout, conf, _context) {
```
Comment (WilcoFiers) [inline index.js]: Most importantly we're going to need to write a little documentation for this. If you can put something in the readme that would be enough.

**Comment by jkodu** [comment]
@J-Kallunki the CI failure is fixed on the base branch. Could you please update your branch.

**Comment by WilcoFiers** [comment]
Thank you for your contribution @J-Kallunki!

**Comment by neomi2428** [comment]
@WilcoFiers I am looking for this feature. Would you let me know when this change would be released? I just want to know rough release plan. Thanks for building this cool project!

**Comment by straker** [comment]
@neomi2428 We've just created a release for it, should be merged in a few days. https://github.com/dequelabs/react-axe/pull/130

**Comment by neomi2428** [comment]
@straker Sweet! I can't wait. 👍

**Comment by neomi2428** [comment]
@straker I am wondering if you have plan to support a typescript version of react-axe. Do you have plan for that?

**Comment by straker** [comment]
@neomi2428 we currently don't have any plans for typescript, but I'll certainly look into it. I imagine it wouldn't be too hard to support for this repo.

**Comment by neomi2428** [comment]
I found that someone did it for react-axe 3.1. Just in case you want to look into it, I share below links.
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34108/files
- https://github.com/dequelabs/react-axe/issues/73
Thanks!

**Comment by straker** [comment]
Awesome, it's a bit out of date though. But it was helpful for doing it myself.

**Comment by straker** [comment]
@neomi2428 just finished a merge #137 that converted the library to typescript

**Comment by neomi2428** [comment]
That is awesome! Thanks a lot. 👍

**Pull request merged**